lsp/vue_ls.lua

```diff
@@ -41,7 +41,8 @@ return {
           payload,
         },
       }, { bufnr = context.bufnr }, function(_, r)
-        local response_data = { { id, r.body } }
+        local response = r and r.body or nil
+        local response_data = { { id, response } }
```

Comment:
The change you have made does not change anything unless `r.body` is `false` ...?